packages/volto-blocks/src/components/blocks/IconsAndText/View.tsx

```diff
@@ -0,0 +1,75 @@
+import config from '@plone/registry';
+import type { BlockViewProps } from '@plone/types';
+import { TextBlockView } from '@plone/volto-slate/blocks/Text';
+import { UniversalLink } from '@plone/volto/components';
+import ViewItem from '@redturtle/volto-blocks/components/blocks/IconsAndText/ViewItem';
+import type { IconsAndTextData } from '@redturtle/volto-blocks/components/blocks/IconsAndText/schema';
+import styles from '@redturtle/volto-blocks/components/blocks/IconsAndText/styles.module.scss';
+import cx from 'classnames';
+import React from 'react';
+
+type Props = Omit<BlockViewProps, 'data'> & {
+  data: IconsAndTextData;
+};
+
+export default function View({ data, className, style }: Props) {
+  const column_number = data.column_number ? parseInt(data.column_number) : 3;
+
+  const Container = config.getComponent('Container').component || 'div';
+
+  return (
+    <>
+      <section
+        className={cx('block block-icons-text', styles.block_icons_text)}
+        aria-label={data.title}
+      >
+        <Container className={cx('block-iconsandtext-container')}>
+          <div
+            className={cx(
+              styles['block-content-header'],
+              styles['header-align-center'],
+              styles['image-wrapper'],
+            )}
+          >
+            {data.title && (
+              <h2
+                className={cx('block-iconsandtext-title title', styles.title)}
```

Comment:
```suggestion
                className={cx('block-iconsandtext-title', styles.title)}
```
`title` is way too generic for a class here